lib/c_glib/src/thrift/c_glib/processor/thrift_multiplexed_processor.c

```diff
@@ -41,6 +41,8 @@ static GParamSpec *thrift_multiplexed_processor_obj_properties[PROP_THRIFT_MULTI
 static gboolean
 thrift_multiplexed_processor_register_processor_impl(ThriftProcessor *processor, const gchar * multiplexed_processor_name, ThriftProcessor * multiplexed_processor , GError **error)
 {
+  (void)error;
```

Comment:
Thrift already provides a macro for this:
```suggestion
  THRIFT_UNUSED_VAR (error);
```